src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -187,12 +187,22 @@ public T GetDebuggerConfigurationSetting<T>(string settingName, T defaultValue)
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Lookup the clsid of the launcher that understands these options
+        /// </summary>
+        /// <param name="launchOptionsName">launch options type name</param>
+        /// <returns></returns>
+        public Guid GetCustomLauncherClsid(string launchOptionsName)
```

Comment:
Since we need to define a new interface anyway, I would suggest we add 'GetCustomLauncher' (no Clsid) and return the object directly. Since Clsid aren't exsactly a natural concept on non-windows.